tests/conftest.py

```diff
@@ -20,7 +20,10 @@ def pytest_generate_tests(metafunc):
     """Generate the tests."""
     if "operator" in metafunc.fixturenames:
         for callspec in metafunc._calls:
-            op_call = callspec.params["operator"]
+            try:
+                op_call = callspec.params["operator"]
+            except KeyError:
+                continue
```

Comment:
I am not familiar with the tests, can you confirm it is alright to just skip without further consideration ? What would be a case where it happens ?

Reply:
Yes. this check if the operator fixture has parameters, and if one of the  parameters is backend, then we check if the backend is available , otherwise the test is skipped. This all function is there to adapt to the local setup.